test/Spec/State/ForPairs.spec.luau

```diff
@@ -17,7 +17,7 @@ return function()
 		local expect = getfenv().expect
 		
 		local scope = {}
-		local forObject = ForPairs(scope, {}, function()
+		local forObject = ForPairs(scope, {}, function(use, scope, key, value)
 			-- intentionally blank
```

Comment:
Use, Scope, Key, Value shouldn't be provided as it is never used
```suggestion
		local forObject = ForPairs(scope, {}, function()
```

Reply:
reverted